### ISSUE TITLE
Мистический способ идентификации штаммов вирусов

### ISSUE THREAD

**Issue by Sakuya-Izayoi** [TauCetiStation/TauCetiClassic#5742]
1. Вирусы идентифицируются простым экзамайном по вирус дишю. Причём простой экзамайн показывает прям подробную информацию.
2. Если у вируса просканированого в дизиз аналайзере поменять симптомы (можно сделать его вообще другим вирусом) у него останется тот же номер в базе.

**Comment by Tolerantboy** [comment]
Нет, вирусы идентифицируются так только после сканирования, что можно объяснить, что сканер просто написал инфу на банке с вирусом
